src/common/Button/Button.jsx

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import styles from './styles.module.scss';
 
-const Button = ({ buttonText, showCourseButtonStyle, onClick }) => (
+const Button = ({ buttonText, showCourseButtonStyle, onClick, type }) => (
 	<button
 		className={showCourseButtonStyle ? showCourseButtonStyle : styles.button}
 		onClick={onClick}
+		type={type}
 	>
 		{buttonText}
 	</button>
 );
 
+Button.propTypes = {
+	buttonText: PropTypes.string,
```

Comment:
buttonText is used in every Button those it should be required prop. `PropTypes.string.isRequired`
For not required props define default value, for example:
`Button.defaultProps = {
    type: 'button'
}`

Reply:
Do it for all components with proptypes